cloud/pkg/cloudhub/servers/httpserver/certutil.go

```diff
@@ -60,14 +61,28 @@ func NewSelfSignedCACertDERBytes(key crypto.Signer) ([]byte, error) {
 }
 
 func NewCloudCoreCertDERandKey(cfg *certutil.Config) ([]byte, []byte, error) {
-	serverKey, _ := NewPrivateKey()
-	keyDER, _ := x509.MarshalECPrivateKey(serverKey.(*ecdsa.PrivateKey))
+	serverKey, err := NewPrivateKey()
+	if err != nil {
+		klog.Errorf("Failed to generate a privateKey, err: %v", err)
+	}
+
+	keyDER, err := x509.MarshalECPrivateKey(serverKey.(*ecdsa.PrivateKey))
+	if err != nil {
+		klog.Errorf("Failed to convert an EC private key to SEC 1, ASN.1 DER form, err: %v", err)
+	}
 
 	// get ca from config
 	ca := hubconfig.Config.Ca
-	caCert, _ := x509.ParseCertificate(ca)
+	caCert, err := x509.ParseCertificate(ca)
+	if err != nil {
+		klog.Errorf("Failed to parse a caCert from the given ASN.1 DER data, err: %v", err)
+	}
+
 	caKeyDER := hubconfig.Config.CaKey
-	caKey, _ := x509.ParseECPrivateKey(caKeyDER)
+	caKey, err := x509.ParseECPrivateKey(caKeyDER)
+	if err != nil {
+		klog.Errorf("Failed to parse ECPrivateKey, err: %v", err)
+	}
```

Comment:
We need return all these errors to `SignCerts` , and in `SignCerts` return to `PrepareAllCerts`, and finally call the klog.Fatel to exit cloudcore.

And delete the `fmt.Printf("%v", err)` in SignCerts , return the err